setup.cfg

```diff
@@ -48,6 +48,8 @@ branch = True
 cover_pylib = False
 source =
     coalib
+omit =
+    tests/bears/DebugBearsTest.py
```

Comment:
Why do you exclude the test file? Its coverage isn't measured anyway^^

Reply:
tests are by default omitted for coverage, measuring test coverage usually doesn't make sense :P

Reply:
test coverage of tests* :P